Division codigo en modulos

maurperez opened the pull request.

Commit:
Instalacion ESlint

Commit:
Division del codigo en modulos

Division aun inclompleta

Commit:
Codigo divido en modulos

La pagina ya funciona con la division del codigo en modulos. Falta corregir ciertos nombre y ver que la ubicacion de cada funcion sea la adecuada

Commit:
Ultimas correciones

Commit:
Agrega unidades de medidas para los pokemons

Commit:
Eliminacion antiguo index

Tambien se corrigio un pequeño error con la ultima URL

fsodano commented on clase-12/tarea/tarea-pokeAPI/src/index.js:
```diff
@@ -1,212 +1,74 @@
+
+const URLprimera = 'https://pokeapi.co/api/v2/pokemon/?limit=40'
+
+let CantidadTotalPokemons
```
cantidadTotalPokemons

maurperez replied:
Se me escapo ese

fsodano commented on clase-12/tarea/tarea-pokeAPI/src/index.js:
```diff
@@ -1,212 +1,74 @@
-    })
-
-
+async function clickCarta () {
```
por qué esta función es async? Y además, clickCarta me da a entender que le estás haciendo click a una carta... en realidad debería llamarse `definirManejadorClickCarta` o algo similar a eso

maurperez replied:
Ese async se me escapo :sweat_smile:
Por lo de las funciones, ahi lo arregle.

fsodano commented on clase-12/tarea/tarea-pokeAPI/src/index.js:
```diff
@@ -1,212 +1,74 @@
-    mostrarInfoPokemon(nombrePokemon)
-
+$('#padre-anterior').click((e) => {
+  if (testURLdañada(URLanterior, URLsiguiente)) {
```
qué hace testURLDañada? (no le pongas ñ a las variables ni funciones, de paso)

maurperez replied:
Eso fue uno de los principales problemas que tuve con la pokedex. Yo coloque 40 pokemons por pag, y en total son 964. Cuestión al llegar a la ultima pagina únicamente muestra 4 pokemons, eso esta bien, pero el problema surge cuando queres volver para la pagina anterior. Esa ultima pagina me devolvía una URLprevious con un limit=4 y me empezaba a mostrar solamente 4 pokemons por pagina.
Lo que hice fue que antes de cargar la URL, pase por un test que verificara el limit, si no lo pasa. llama a una función que devuelve la url ya corregida.

fsodano reviewed (changes requested):
la división está bien y el código está bastante claro, fijate por favor que todo esté en camelCase y no PascalCase y avisame y lo chusmeo otra vez!

fsodano commented:
Ah, y las funciones tienen que tener un verbo en infinitivo adelante, siempre. Es decir "calcularSalarioAnual" y no "salarioAnual".

Commit:
Correciones superficiales

Correcion de nombres de funciones, elimanacion de las "ñ", eliminacion de una async innecesario
